Codigo/Declares.bas

```diff
@@ -2764,3 +2763,16 @@ End Sub
 Public Sub IncreaseSingle(ByRef dest As Single, ByVal amount As Single)
     dest = dest + amount
 End Sub
+
+Public Sub PerformanceTestStart(ByRef Timer As Long)
```

Comment:
I'd not add this new code to measure time as we already have the class `Instruments` as used in
https://github.com/ao-org/argentum20-server/blob/master/Codigo/UnitTesting.bas#L86


See https://github.com/ao-org/argentum20-server/blob/master/Codigo/Instrumentation.cls